packages/failurechecker/package.json

```diff
@@ -27,8 +27,7 @@
   },
   "dependencies": {
     "gcf-utils": "1.6.1",
-    "moment": "^2.24.0",
-    "@octokit/rest": "16.41.2"
```

Comment:
if we're going to remove `@octokit/rest`, it should be added as a dependency to `gcf-utils`.

Reply:
Oh I like that idea.  I thought it already was!

Reply:
It already was :)